src/instructlab/model/download.py

```diff
@@ -80,8 +80,10 @@ def download(self):
             if self.ctx.obj is not None:
                 hf_logging.set_verbosity(self.ctx.obj.config.general.log_level.upper())
             files = list_repo_files(repo_id=self.repository, token=self.hf_token)
-            if any(".safetensors" in fname for fname in files):
-                self.download_safetensors()
+            if any(".safetensors" in fname for fname in files) or any(
+                ".bin" in fname for fname in files
+            ):
```

Comment:
```suggestion
         if any(re.search(r"\.(safetensors|bin)$", fname) for fname in files):
```

Reply:
regexp makes me sad but I will use this :laughing: